src/schemas/version-lock.description.md

```diff
@@ -0,0 +1,3 @@
+The `version-lock` property allows pinning of specific services to a particular version number.
+
+Each property is the name of the service, the value is either the version number or (when starting with `v`) or the CI build number (when starting with `ci`)
```

Comment:
the current version lock header is way more flexible. as it allows complete action name replacement, eg:

```
gdocs2md=gdocs-test@1.0
```
(although of questionable use :-)

if only the versions can be modified, I would allow all characters and not limit it with `v*` and `c*` patterns.

Reply:
Ok, I'll relax the schema